sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -1116,6 +1119,41 @@ public ContactDirectorySteps(
               CONTACT_DIRECTORY_DETAILED_PAGE_APPLY_FILTER_BUTTON);
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
+
+    Then(
+        "I set vaccination contact status filter to ([^\"]*) and apply",
+        (String vaccinationStatus) -> {
+          webDriverHelpers.selectFromCombobox(
+              CONTACT_VACCINATION_STATUS_FILTER_COMBOBOX, vaccinationStatus);
+          TimeUnit.SECONDS.sleep(2);
+
+          webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTERS_BUTTON);
+        });
+
+    Then(
+        "I set contact vaccination status filter to ([^\"]*) and apply",
+        (String vaccinationStatus) -> {
+          webDriverHelpers.selectFromCombobox(
+              CONTACT_VACCINATION_STATUS_FILTER_COMBOBOX, vaccinationStatus);
+          webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTERS_BUTTON);
```

Comment:
move out apply filters logic, so we can have a reusable method (make sure if it doesn't already exist)

Reply:
@rdutu-vg 
Should I create method like webDriverHelpers.selectAndApplyFilter(field, value) or selectFilter(field, value) and applyFilter()?